### DISCUSSION TITLE
HTML to Markdown: Unsupported charset WINDOWS-1251

### DISCUSSION THREAD

**Discussion by johnmapeson** [jgm/pandoc#10104] [Q&A] [answered]
Pandoc 3.3

I try to convert the translation of the Jerome K. Jerome's "Three Men in a Boat" (from the webpage of the guy who translated it) from HTML to Markdown.

<https://www.gaisever.net/miscellanea/three_men_in_a_boat.html>

```
pandoc -f html -t markdown -o output.md https://www.gaisever.net/miscellanea/three_men_in_a_boat.html
```

When I do so, there is an error:

> Unsupported charset WINDOWS-1251

Is there a way to workaround it?

**Comment by jgm** [comment, marked as answer]
First use `iconv` to convert the file to UTF-8.
```
iconv -f=WINDOWS-1252 -t=UTF-8
```

**Reply by johnmapeson** [reply]
`iconv -f WINDOWS-1252 -t UTF-8` works for me, thanks.
